### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by tj** [apex/go-apex#5]

**Comment by harlow** [comment]
@tj are you thinking an example for each of the data source types? i.e. Kinesis, S3, SNS, etc?

**Comment by tj** [comment]
yep! that would be nice

**Comment by harlow** [comment]
Was thinking about  a Kinesis example. How do you feel about golang function that accepts records from stream and sends the messages to requestb.in so there is an `in -> out` component to the example.

**Comment by tj** [comment]
sounds good to me!

**Comment by harlow** [comment]
@tj is it possible to set ENV vars through Apex in Lambda? I'd like to be able to initialize some variables in lambda function (independent of the data coming from the stream). 

I've seen some examples of using tags (metadata) from the stream itself. Not sure if apex supports pulling that data yet.

**Comment by tj** [comment]
yup! either via `environment: {}` in the project/function json or `--set foo=bar` with the cli
